src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -147,7 +147,7 @@ const getSignUpLink = (appState, settings) => {
   if (appState.hasRemediationObject(RemediationForms.SELECT_ENROLL_PROFILE)) {
     const signupLinkData = {
       'type': 'link',
-      'label': loc('signup', 'login'),
+      'label': loc('oie.registration.form.title', 'login'),
```

Comment:
@sherizada-okta - for customers on the v1 pipeline who have self-service registration, is this going to break their i18n customizations? I'd assume we should be looking to reuse this [key](https://github.com/okta/okta-signin-widget/blob/dfd802db8ac36bca4f5873e16251801afd7b9648/src/views/shared/FooterRegistration.js#L46) instead of creating a new one.